src/main/java/org/folio/circulation/domain/policy/LoanPolicy.java

```diff
@@ -363,6 +363,10 @@ LoanPolicy withAlternateRenewalSchedules(JsonObject renewalSchedules) {
     return withAlternateRenewalSchedules(FixedDueDateSchedules.from(renewalSchedules));
   }
 
+  public boolean isLoanable() {
```

Comment:
This can be used in the `isNotLoanable` method, to improve readability and reduce duplication

Reply:
Done